testing/functional/tests/cypress/page-objects/student-objects/WelcomeObject.ts

```diff
@@ -0,0 +1,63 @@
+export default class WelcomeObject {
+
+    checkCurrentUrl() {
+        cy.url().should('include', 'https://test-aest-sims.apps.silver.devops.gov.bc.ca/student')
+        cy.wait(2000)
```

Comment:
I still can see a lot of `cy.wait`, as mentioned before it seems to be a bad practice as stated below:
https://docs.cypress.io/api/commands/wait#Time

Can you check the proper way to handle this on Cypress, maybe below link helps.
https://docs.cypress.io/faq/questions/using-cypress-faq#How-can-I-make-Cypress-wait-until-something-is-visible-in-the-DOM

Reply:
https://github.com/bcgov/SIMS/pull/1089#discussion_r834697506